deploy-image/.aea/cli_config.yaml

```diff
@@ -1,3 +0,0 @@
-auth_token: <your-auth-token>
-author: <your-username>
-registry_path: "packages"
```

Comment:
We use IPFS registry by default now so we initialize the remote when building the docker image